### ISSUE TITLE
Impossible to run custom NPM scripts

### ISSUE THREAD

**Issue by Anahkiasen** [phpro/grumphp#250]
| Q               | A
| --------------- | ---
| Branch          | 0.11.0 
| Bug?            | yes
| New feature?    | no
| Question?       | no
| Documentation?  | no
| Related tickets | 

**My configuration**
```yaml
parameters:
    git_dir: .
    bin_dir: ./vendor/bin
    tasks:
        npm_script:
            script: run lint
            triggered_by: [js]
            working_directory: "./"
```

**Steps to reproduce:**

Run Grumphp

**Result:**

In NPM, you can add custom scripts to the `package.json` file. Custom being any scripts that are not core scripts (npm build, npm test, etc). To run those, you call `npm run {name of script}`.

Issue is currently Grumphp runs `npm 'run lint'` instead of `npm run lint`, which fails.

**Comment by veewee** [comment]
Thanks @Anahkiasen for reporting this issue.
The script parameter is indeed handled as one argument.
Possible solutions would be:

- Create an additional `is_custom` or `is_run_task` parameter which will add the run before the `script` parameter in the CLI.
- Create an additional `arguments` parameter in which you can specify custom cli arguments.
- ... ?

Can you tell me which you like best?

**Comment by Anahkiasen** [comment]
Could Grumphp recognize if the custom script starts with `"run "`, it's always either going to or not going to anyway no?

**Comment by veewee** [comment]
That is also a possibility. However, I prefer to keep the configuration explicit. This way, the parameters can't be abused and won't contain any unpredictable logic. Maybe the boolean option is the best choice. As stated in the NPM docs, it is also possible to add custom attributes to a custom command. We could use an arguments parameter for that case.

**Comment by veewee** [comment]
This issue is fixed in #251. Can you check if this is working for you?
